website/versioned_docs/version-0.10.1/features/lightgbm/about.md

```diff
@@ -57,6 +57,48 @@ model = LightGBMRegressor(application='quantile',
 For an end to end application, check out the LightGBM [notebook
 example](../LightGBM%20-%20Overview).
 
+### Arguments
+
+SynapseML exposes getters/setters for many common LightGBM parameters.
+In python, you can use the properties as shown above, or in Scala use the
+fluent setters.
+```scala
+import com.microsoft.azure.synapse.ml.lightgbm.LightGBMClassifier
+val classifier = new LightGBMClassifier()
+                       .setLearningRate(0.2)
+                       .setNumLeaves(50)
+```
+
+LightGBM has far more parameters than SynapseML exposes. For cases where you
+need to set some parameters that SyanpseML does not expose a setter for, use
+passThroughArgs. This is just a free string that you can use to add extra parameters
+to the command SynapseML sends to configure LightGBM.
+
+```python
+from synapse.ml.lightgbm import LightGBMClassifier
+model = LightGBMClassifier(passThroughArgs="force_row_wise=true",
```

Comment:
maybe provide example of passing more than one args in python as well? Similar to how it is done with Scala example. Passing argument with whitespace as a Seperator might not be intuitive. 

model = LightGBMClassifier(passThroughArgs="force_row_wise=true min_sum_hessian_in_leaf=2e-3",

Reply:
Great idea @JessicaXYWang !